docker-compose-separate-containers.yml

```diff
@@ -0,0 +1,34 @@
+version: '2'
+services:
+  nginx:
+    image: nginx
+    container_name: nginx
+    ports:
+      - "80:80"
+    volumes:
+      - /tmp/nginx:/etc/nginx/conf.d
```

Comment:
I'm removing mounting /tmp on the host. Suggesting to mount a static path on the host when not necessary seems like a recipe for a confusing situation where two docker-gen's are writing to the same volume.